core/lib/dal/src/fri_witness_generator_dal.rs

```diff
@@ -69,7 +72,7 @@ impl FriWitnessGeneratorDal<'_, '_> {
         last_l1_batch_to_process: u32,
```

Comment:
please add comment to the method, and explain what it returns - especially the Vec<u8> part (is it raw pubdata ? ome blob ? empty)